src/collects/seashell/backend/dispatch.rkt

```diff
@@ -451,7 +451,7 @@
         (_ _) ...)
        (define checksum
          (write-file project file (string->bytes/utf-8 contents)
-                     (if history (string->bytes/utf-8 history) "")
+                     (if history history "")
```

Comment:
I remember the issue now - json-structs could have strings, but not bytes - so previously I just changed history to be a string *everywhere*. I went back and re-did it so that history is converted to a string for the json-structs, but kept as bytes everywhere else.